playwright/transform.js

```diff
@@ -15,6 +15,6 @@ module.exports = {
         '@babel/preset-react',
       ],
     });
-    return result ? result.code : src;
+    return result ? { code: result.code } : src;
```

Comment:
Is this a breaking change from Jest 28? I was really hoping we could support both Jest versions still

Reply:
Yes this is a breaking change to support jest 28. See the [note](https://jestjs.io/docs/upgrading-to-jest28#transformer) about the Transformer having to now be an object instead of a string.

Reply:
We would have to check what version of Jest is used and then change the return value.

Reply:
@yannbf I've added a check for the version of Jest used and then change the return.